### PR TITLE
Add SharedRuntime1.1 to Excel

### PR DESCRIPTION
**Pull request by shaofengzhu**
Add SharedRuntime1.1 to Excel

### CODE REVIEW

File: mapping/requirements_officejs.json [hunk 1]
```diff
@@ -980,7 +980,18 @@
...
 							}],
 							"availability": "GA"
-						}                        
+						},
+						{
+							"name": "SharedRuntime",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.35",
```
Comment (sumurthy) [inline mapping/requirements_officejs.json:989]: We need to use Telemetry Version number such as 16.37.0.0 (default last 2 to zero). Please check MRO site to figure out the exact ver# for MacOS and iOS.

Comment (mscharlock) [reply mapping/requirements_officejs.json:989]: Ok folks, I have to bed for the night soon. I will keep checking this for another hour or so. If it's after 10:30PM PST, I won't be on to make this change for you in the docs. What you'll need to do then is make a pr to the office-js-docs-pr repo with the new build number ([listed on the Shared Runtime requirement set page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/shared-runtime-requirement-sets)). You will need to push your branch into the master branch and then master into live. If you don't have the ability to do that for some reason, let me know in the morning before the 8 AM PST call so I can push it out. (Linda Lu or any member of the docs team should also be able to push to live as well, FYI).

Comment (shaofengzhu) [reply mapping/requirements_officejs.json]: Update using MAC telemetry version 16.34.209.0

**Comment by sumurthy** [comment]
Is this not supported on iOS?

File: mapping/requirements_officejs.json [hunk 2]
```diff
@@ -980,7 +980,18 @@
...
 							}],
 							"availability": "GA"
-						}                        
+						},
+						{
+							"name": "SharedRuntime",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.34.209.0",
```
Comment (sumurthy) [inline mapping/requirements_officejs.json:989]: Please default last 2 to zeros - 16.34.0.0

Comment (mscharlock) [inline mapping/requirements_officejs.json:989]:
```suggestion
									"build": "16.34.0.0",
```

**Comment by sumurthy** [comment]
Unexplained Merge Conflict - hence closing it.

**Pull request closed without merging**